libs/partners/anthropic/langchain_anthropic/chat_models.py

```diff
@@ -677,9 +672,15 @@ def post_init(self) -> Self:
         if self.default_request_timeout is None or self.default_request_timeout > 0:
             client_params["timeout"] = self.default_request_timeout
 
-        self._client = anthropic.Client(**client_params)
-        self._async_client = anthropic.AsyncClient(**client_params)
-        return self
+        return client_params
+
+    @cached_property
+    def _client(self) -> anthropic.Client:
```

Comment:
if we don't have it at top-level, it doesn't matter, and also removes a validator